runtime/runtime-params-estimator/test-contract/build.sh

```diff
@@ -46,8 +46,11 @@ echo ${bare_wasm}
 # Generate several files of various sizes. We will compile these files into the Wasm binary to
 # bloat its size to the given values.
 
+# FIXME(#6822): we should just remove the payload logic. 10Kib variant is
+# broken, because the baseline contract is >10KiB (data for alt_bn estimatons).
+
 # 10KiB
-dd if=/dev/urandom of=./res/payload bs=$(expr 10240 - ${bare_wasm}) count=1
+dd if=/dev/urandom of=./res/payload bs=1 count=1
 cargo build --target wasm32-unknown-unknown --release  --features "payload$features_with_comma"
 cp target/wasm32-unknown-unknown/release/test_contract.wasm ./res/stable_small_contract.wasm
```

Comment:
Why FIXME? Can’t we just do it now? If bare_wasm is ≥ 10240 than just `cp -- test_contract.wasm stable_small_cotract.wasm` and we can go on with our lives.

Reply:
>If bare_wasm is ≥ 10240 than just cp -- test_contract.wasm stable_small_cotract.wasm and we can go on with our lives.

The contract will then be bigger than 10KiB, but the current code is written as if it being exactly 10KiB matters. 

Ultimately, I suspect that this whole file is mostly dead code at this point, but I'd rather not deal with it during stabilization PR.

Reply:
Yeah please do not change it in this PR. The estimator uses the sizes. Eventhough it doesn't rely on it being exact, I would still want to check that and a stabilization PR is not the right place for such a change anyway.